### PR TITLE
fix: Travis

### PR DESCRIPTION
**Pull request by liveHarshit**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,9 @@ cache:
       - "$HOME/.gradle/caches/"
       - "$HOME/.gradle/wrapper/"
 install:
-  - echo yes | ${ANDROID_HOME}/tools/bin/sdkmanager "platforms;android-28"
+  - touch $HOME/.android/repositories.cfg
+  - yes | sdkmanager "platforms;android-28"
+  - yes | sdkmanager "build-tools;28.0.3"
```
Comment (ShridharGoel) [inline .travis.yml]: Try reverting this

Comment (liveHarshit) [reply .travis.yml]: Will not work

Comment (liveHarshit) [reply .travis.yml]: > Try reverting this

Circle CI is also failing

**Comment by liveHarshit** [comment]
Problem is with the directory ${ANDROID_HOME} which is: `/usr/local/android-sdk/`

**Pull request closed without merging**